src/content/glossary/index.md

```diff
@@ -602,7 +602,7 @@ A special type of [contract](#smart-contract) that has no payable functions, no
   Smart Contract Libraries
 </DocLink>
 
-### lightweight client {#lightweight-client}
+### light client {#light-client}
```

Comment:
Good call. We use a couple of naming conventions across the site that we should fix eventually.